docs/modules/ROOT/pages/includes/quarkus-log-handler-splunk.adoc

```diff
@@ -153,15 +261,29 @@ a| [[quarkus-log-handler-splunk_quarkus.log.handler.splunk.metadata-host]]`link:
 [.description]
 --
 Overrides the host name metadata value.
+
+ifdef::add-copy-button-to-env-var[]
+Environment variable: env_var_with_copy_button:+++QUARKUS_LOG_HANDLER_SPLUNK_METADATA_HOST+++[]
+endif::add-copy-button-to-env-var[]
+ifndef::add-copy-button-to-env-var[]
+Environment variable: `+++QUARKUS_LOG_HANDLER_SPLUNK_METADATA_HOST+++`
+endif::add-copy-button-to-env-var[]
 --|string 
-|`The equivalent of %h in a formatted message`
+|
```

Comment:
I think this is a bug in our default value generation. I'll open an issue and fix it so that it gets properly generated in future upgrades.